libpod/container_internal.go

```diff
@@ -1042,6 +1042,91 @@ func (c *Container) cniHosts() string {
 	return hosts
 }
 
+// Run networksetup hooks
+func (c *Container) runNetworkSetupHooks(ctx context.Context) error {
```

Comment:
Can you refactor this to share code with `setupOCIHooks`? They look largely identical.

Reply:
Done!